Cargo.toml

```diff
@@ -11,18 +11,22 @@ thiserror = "1.0.30"
 smallvec = "1.8.0"
 pyo3 = {version = "0.17.1", features = ["extension-module"], optional = true}
 quickcheck = {version = "1.0.3", optional = true}
+serde = {version = "1.0", features = ["derive"], optional = true}
+serde_with = {version = "2", optional = true}
 
 [dev-dependencies]
 criterion = "0.4.0"
 rand = "0.8.5"
+serde_json = "1"
 
 [lib]
 name = "quickdna"
 crate-type = ["cdylib", "rlib"]
 
 [features]
 python-support = ["dep:pyo3"]
-default = ["python-support"]
+serde = ["dep:serde", "dep:serde_with"]
+default = ["python-support", "serde"]
```

Comment:
Should we default-on serde? Most libraries don't.

Reply:
Ah, okay, turning it off is fine by me so I did that :)